### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by UsirusenIkon**
In this pull request, we initialized:
-  linters to the document
-  Changed the header color to grey
-  Updated my readme file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+test.md
```
Comment (cynthiainga) [inline .gitignore:1]: - [ ] Please kindly add `node-modules` to your `.gitignore file` as mentioned in the requirements 🙃 🙃

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
...
+# Hello Microverse
+
+>setting up linters
+
+
+## Built With
+
+- html & css
+- vs code
+
```
Comment (cynthiainga) [inline README.md]: - [ ] Please the README file should be descriptive, kindly add more details in your README file, you can use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) 😊 😊

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @UsirusenIkon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] No linters errors ✔️ 
- [ ] Used the correct Github flow: PR is made from the feature branch into the main branch ✔️ 

## Required Changes ♻️

- [ ] Please the Pull Request should have a summary describing introduced changes in more detail, kindly add it 🙃 🙃 
- [ ] Pull Request should have a descriptive title telling you what changes are inside, for example instead of `feature` you can name it `Hello Microverse project` 😁 😁
- [ ] Please check the comments under the review for other changes required 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- So then if you change your branch name this PR will be closed but in your future project please your branch should have the name of a feature or action (e.g. 'created_sidebar' or 'hello_microverse',...).
- I recommend you to use the imperative mode when you write your commits messages, for example instead of `added color` write `Add color`, is the action that you do in the present, not in the past, and also the first letter is in capital 😁 😁 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @UsirusenIkon   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._
- Kindly make sure to have a descriptive README. Please make sure to use this microverse README [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as requested by the previous reviewer . Clone it and remember to custimze it to make it personal :sunglasses: 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [OPTIONAL] Kindly make sure to write your commit message in imperative tense and capitalized it. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @UsirusenIkon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**